resources/views/profile/show.blade.php

```diff
@@ -37,31 +38,93 @@
+                                @if(isset($user['degree_front']))
+                                    <div>
+                                        Titul před:
+                                        {{$user['degree_front']}}
+                                    </div>
+                                @endif
+                                @if(isset($user['degree_after']))
+                                    <div>
+                                        Titul za:
+                                        {{$user['degree_after']}}
+                                    </div>
+                                @endif
+                                @if(isset($user['school']))
+                                    <div>
+                                        Škola:
+                                        {{$user['school']}}
+                                    </div>
+                                @endif
                             </div>
-                        </div>
                         @endif
                     </div>
                 </div>
-            </div>
-
+                @if($user['id'] != Auth::id())
+                    <div class="card mt-3">
+                        <div class="card-header">
+                            Společné skupiny
+                        </div>
+                        <div id="groups_body" name="groups_body" class="card-body">
 
+                        </div>
+                    </div>
+                @endif
+            </div>
         </div>
     </div>
+
+    <script src="https://code.jquery.com/jquery-3.4.1.min.js"
+            integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
+
+    <script>
+        @if(isset($user['id']) && Auth::id() != $user['id'])
+        getCommonGroups();
+        @endif
+
+        function getCommonGroups() {
+            $.post('{{ route("profile.commonGroups") }}',
+                {
+                    _token: $('meta[name="csrf-token"]').attr('content'),
+                    user_id: {{$user['id']}}
+                },
+                function (data) {
+                    console.log(data);
+                    postCommonGroups(data);
+                });
+        }
+
+        function postCommonGroups(data) {
+            htmlView = '<div class="row gap-3 m-2 d-flex flex justify-content-evenly align-items-start">';
+            for (let i = 0; i < data.result.length; i++) {
+                htmlView += `
+                        <div class="card p-0 me-auto" style="width: 18rem;">
+                            <img src="` + data.result[i].photo + `" class="card-img-top" style="height: 215px; width: calc(inherit - 1);" alt="Foto skupiny">
+                            <div class="card-body">
+                                <h5 class="card-title" style="text-overflow: ellipsis; white-space: nowrap; overflow: hidden" title="` + data.result[i].name + `">` + data.result[i].name + `</h5>
```

Comment:
`style="text-overflow: ellipsis; white-space: nowrap; overflow: hidden"` makes it so long text is wrapped and ends with three dots


`title="+ data.result[i].name + "` makes it so the wrapped text is shown upon mouse hover